presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -113,6 +113,11 @@
 
     boolean isImpersonationEnabled();
 
+    default Optional<String> getConfigValue(String name)
```

Comment:
Please add this to the previous commit that adds the other methods to the interface. (same for `ThriftMetastore`)

Or maybe just squash that commit into this one, since it is tiny and all the real work is done here.

Reply:
Merged with previous commit.